code/modules/mob/living/carbon/human/species.dm

```diff
@@ -703,6 +736,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 
 /datum/species/proc/spec_life(mob/living/carbon/human/H)
+	if(H.has_trait(TRAIT_TOXIMMUNE))
+		H.setToxLoss(0)
```

Comment:
what the hell is this

Reply:
Aww fuck, it's leftover. I apologize for that.